syntax = "proto3";

package echo;

message TransmissionObject {
    float value = 1;
    string message = 2;
}